src/index.html

```diff
@@ -60,31 +60,31 @@
 
     <main class="catalog">
       <div class="catalog__card">
-        Product card
+        <p class="card-title">Product card</p>
```

Comment:
You don't need to change the html file. Just use the correct styles. Discard all changes in this file, please